src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,100 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2 / span 2;
```

Comment:
You can't actually set this here, since this will cause the inner grid to be misaligned. If you have the inner grid span the outer grid's center and right bleed areas, then the inner grid will evenly divide that space into 12, which means that it'll be bigger than an inner grid that is only in the center area.

Since you have the black background out in the right bleed area, you probably need to go with the approach that we've taken in a few of the other components with asymmetric bleeds, such as the TwoParagraphBlock, where we create a separate `<div>` whose job is to paint the background: https://github.com/ShelterTechSF/sheltertech.org/blob/b7c96e585126cb9c142064a18008ce1debfe1dc3/src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css#L10-L14

Note that this also has the side effect of causing the mobile view to lack any right padding, which is noticeable on a small phone screen (320px):

<img width="417" alt="Screen Shot 2020-11-15 at 5 15 50 PM" src="https://user-images.githubusercontent.com/1002748/99203073-3d170b80-2766-11eb-9284-28f851e928bf.png">

Reply:
👍 Fixed with Richard's assistance! Had to create a new bleedBackground wrapper with row property.